runtime/doc/eval.txt

```diff
@@ -8927,6 +8927,18 @@ settagstack({nr}, {dict} [, {action}])			*settagstack()*
 			let newtag = [{'tagname' : 'mytag', 'from' : pos}]
 			call settagstack(2, {'items' : newtag}, 'a')
 
+<		    Write a jump to the tag stack: >
```

Comment:
As this is a detailed example, can you move this to the tagsrch.txt file
under the tagstack section? You can add a tagstack-examples tag.
You can also move some of the other existing examples to this
section and then refer to this tag from settagstack() in eval.txt.

Reply:
Okay. I picked two to keep in eval.txt and moved the rest.